world_universities_and_domains.json

```diff
@@ -47490,7 +47490,7 @@
       "web_pages": [
           "https://www.iutripura.edu.in/"
       ],
-      "name": "ICFAI University",
```

Comment:
Added the campus state, as added in other campuses, so the name does not conflict with the head campus shown below:
```json
{
      "web_pages": [
          "http://www.icfai.org/"
      ],
      "name": "ICFAI University",
      "alpha_two_code": "IN",
      "state-province": null,
      "domains": [
          "icfai.org"
      ],
      "country": "India"
  },
 ```